<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>399</width>
    <height>700</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>56</red>
        <green>58</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 58, 89);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/Register User Icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nombre">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  NOMBRE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_apellido">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> APELLIDO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_dni">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solidrgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);
}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string> DNI</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="de_fnaci">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>56</red>
          <green>58</green>
          <blue>89</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solidrgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);
}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}
</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_direccion">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> DIRECCIÓN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_email">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> EMAIL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_telefono">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solidrgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);
}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string> TELEFONO </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FECHA DE NACIMIENTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_crear_cuenta">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>660</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 8px;
	background-color: rgb(185, 212, 248);
	color: rgb(255, 255, 255);
}

QPushButton:hover{
	
	background-color: rgb(101, 155, 219);
}</string>
    </property>
    <property name="text">
     <string>CREATE ACCOUNT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_clave">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(38, 38, 48);
	border -radius: 15px;
	color: #FFF;
	
	background-color: rgb(36, 36, 36);

}

QLineEdit:hover{
	border : 2px solid rgb(48,50,62);
 
}

QLineEdit:focus{
	border: 2px solid (35,218,233);
	background-color: rgb(47,47,47)
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string> PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_error">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
